dev/how-to/maintain/identify-slow-queries.md

```diff
@@ -76,33 +78,33 @@ tidb > show create table INFORMATION_SCHEMA.SLOW_QUERY;
 | Table      | Create Table                                                |
 +------------+-------------------------------------------------------------+
 | SLOW_QUERY | CREATE TABLE `SLOW_QUERY` (                                 |
-|            |   `Time` timestamp UNSIGNED NULL DEFAULT NULL,              |
-|            |   `Txn_start_ts` bigint(20) UNSIGNED DEFAULT NULL,          |
+|            |   `Time` timestamp unsigned NULL DEFAULT NULL,              |
+|            |   `Txn_start_ts` bigint(20) unsigned DEFAULT NULL,          |
 |            |   `User` varchar(64) DEFAULT NULL,                          |
-|            |   `Conn_ID` bigint(20) UNSIGNED DEFAULT NULL,               |
-|            |   `Query_time` double UNSIGNED DEFAULT NULL,                |
-|            |   `Process_time` double UNSIGNED DEFAULT NULL,              |
-|            |   `Wait_time` double UNSIGNED DEFAULT NULL,                 |
-|            |   `Backoff_time` double UNSIGNED DEFAULT NULL,              |
-|            |   `Request_count` bigint(20) UNSIGNED DEFAULT NULL,         |
-|            |   `Total_keys` bigint(20) UNSIGNED DEFAULT NULL,            |
-|            |   `Process_keys` bigint(20) UNSIGNED DEFAULT NULL,          |
+|            |   `Conn_ID` bigint(20) unsigned DEFAULT NULL,               |
+|            |   `Query_time` double unsigned DEFAULT NULL,                |
+|            |   `Process_time` double unsigned DEFAULT NULL,              |
+|            |   `Wait_time` double unsigned DEFAULT NULL,                 |
+|            |   `Backoff_time` double unsigned DEFAULT NULL,              |
+|            |   `Request_count` bigint(20) unsigned DEFAULT NULL,         |
+|            |   `Total_keys` bigint(20) unsigned DEFAULT NULL,            |
+|            |   `Process_keys` bigint(20) unsigned DEFAULT NULL,          |
 |            |   `DB` varchar(64) DEFAULT NULL,                            |
 |            |   `Index_ids` varchar(100) DEFAULT NULL,                    |
-|            |   `Is_internal` tinyint(1) UNSIGNED DEFAULT NULL,           |
+|            |   `Is_internal` tinyint(1) unsigned DEFAULT NULL,           |
 |            |   `Digest` varchar(64) DEFAULT NULL,                        |
 |            |   `Query` varchar(4096) DEFAULT NULL                        |
```

Comment:
You missed Line 92 - Line 101 in https://github.com/pingcap/docs-cn/pull/1278/files.

Reply:
Addressed, thanks for reminding me about it.